app/models/form/sales/subsections/household_situation.rb

```diff
@@ -12,6 +12,16 @@ def pages
       Form::Sales::Pages::LastAccommodation.new(nil, nil, self),
       Form::Sales::Pages::LastAccommodationLa.new(nil, nil, self),
       Form::Sales::Pages::BuyersOrganisations.new(nil, nil, self),
-    ]
+      buyer_2_situation_pages,
+    ].flatten.compact
```

Comment:
not a suggestion but thought this might be inefficient, turns out it's fine, interesting graph: https://stackoverflow.com/questions/34400670/flatten-and-compact-an-array-more-efficiently